README.md

```diff
@@ -1,164 +1,134 @@
-# OpenPubkey Reference Implementation
+# OpenPubkey
 
-This repo contains the current reference implementation of OpenPubkey. The reference implementation is a work in progress.
+## Overview
 
-## What is OpenPubkey?
+OpenPubkey is a protocol for leveraging OpenID Providers (OPs) to bind identities to public keys. It adds user- or workload-generated public keys to [OpenID Connect (OIDC)](https://openid.net/developers/how-connect-works/), enabling identities to sign messages or artifacts under their OIDC identity. 
 
-OpenPubkey adds user or workload generated public keys to OpenID Connect (OIDC) enabling identities to sign messages or artifacts under their OIDC identity. In essence, OpenPubkey is a protocol for getting OpenID Providers (OPs) to bind identities to public keys.
+We represent this binding as a PK Token. This token proves control of the OIDC identity and the associated private key at a specific time, as long as a verifier trusts the OP. Put another way, the PK Token provides the same assurances as a certificate issued by a Certificate Authority (CA) but critically, does not require adding a CA. Instead, the OP fulfills the role of the CA. This token can be distributed alongside signatures in the same way as a certificate.
 
-Verifiers can check that these signatures are valid and associated with the signing OpenID identity. OpenPubkey does not add any new trusted parties beyond what is required for OpenID Connect. It is fully compatible with existing OpenID Providers (Google, Azure/Microsoft, Okta, OneLogin, Keycloak) without any changes to the OpenID Provider.
+OpenPubkey does not add any new trusted parties beyond what is required for OpenID Connect. It is fully compatible with existing OpenID Providers (Google, Azure/Microsoft, Okta, OneLogin, Keycloak) without any changes to the OpenID Provider.
 
-### ID Tokens and PK Tokens
+OpenPubkey can be used to enable a myriad of applications. For example, companies building on OpenPubkey include:
```

Comment:
Can we cut this sentence? It's correct, flows and reasonable but I want the readme overview to be as concise as possible and it doesn't add anything.

Not a blocker, this PR can be merged without cutting this.

Reply:
```suggestion
For example, companies building on OpenPubkey include:
```

Reply:
Fixed